PCbuild/readme.txt

```diff
@@ -14,7 +14,7 @@ Building Python using Microsoft Visual C++
 ------------------------------------------
 
 This directory is used to build CPython for Microsoft Windows NT version
-6.0 or higher (Windows Vista, Windows Server 2008, or later) on 32 and 64
+6.3 or higher (Windows 8.1/Windows Server 2012 or later) on 32- and 64-
 bit platforms.  Using this directory requires an installation of
 Microsoft Visual Studio 2017 (MSVC 14.1) with the *Python workload* and
```

Comment:
This is another section that is just permanently out of date. Can we simplify it to just "Windows" and say "install the latest"? The build should prompt with a fairly clear error message if you happen to get a newer release that we haven't updated it to detect. But it should be basically backwards-forwards compatible with all supported releases - trying to list it all here is just going to make people think it won't work with what they already have.

Reply:
Yeah, that definitely makes more sense long term. I elided the details regarding the specific VS/MSVC versions here, and also simplified the Windows support declaration to avoid duplicating the existing mention in the `using/windows` docs and in the installer and build files (given they also got out of sync).